go.mod

```diff
@@ -69,6 +69,7 @@ require (
 	github.com/leonklingele/grouper v1.1.1
 	github.com/lufeee/execinquery v1.2.1
 	github.com/macabu/inamedparam v0.1.2
+	github.com/maranqz/go-factory-lint v1.0.3
```

Comment:
Could you please fix typos in README.md and the repo's description?

<img width="1289" alt="image" src="https://github.com/golangci/golangci-lint/assets/3228886/33bda8c5-6355-420c-9a08-2a935cb5fa63">

Also, the Go code contains a typo:
```
./factory.go:84: visiter ==> visitor, visit
```

Reply:
@alexandear can you open an issue on the repo instead of adding a comment inside the PR?